pkgs/development/python-modules/facedancer/default.nix

```diff
@@ -2,36 +2,53 @@
   lib,
   buildPythonPackage,
   fetchPypi,
-  isPy3k,
+  pythonOlder,
   pyusb,
   pyserial,
+  prompt-toolkit,
+  libusb1,
+  setuptools,
 }:
 
 buildPythonPackage rec {
   pname = "facedancer";
-  version = "2019.3.2";
-  format = "setuptools";
+  version = "3.0.4";
+  pyproject = true;
+  disabled = pythonOlder "3.8";
 
   src = fetchPypi {
```

Comment:
It's better to build this from source, see https://github.com/greatscottgadgets/facedancer